roles/openshift_rhv/defaults/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Defaults for oVirt/RHV provider integration
+compatibility_version: 4.2
+data_center_name: Default
+wait_for_ip: true
+openshift_rhv_cluster: Default
+template_name: openshift_node
+
+# oVirt VM Profiles
+openshift_rhv_vm_profile:
+  master:
+    cluster: "{{ openshift_rhv_cluster }}"
+    template: "{{ template_name }}"
+    memory: 16GiB
+    cores: 2
+    high_availability: true
+    disks:
+    - size: 15GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: docker_disk
+      interface: virtio
+    - size: 30GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: localvol_disk
+      interface: virtio
+    - size: 25GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: etcd_disk
+      interface: virtio
+    state: running
+  node:
+    cluster: "{{ openshift_rhv_cluster }}"
+    template: "{{ template_name }}"
+    memory: 8GiB
+    cores: 2
+    high_availability: true
+    disks:
+    - size: 15GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: docker_disk
+      interface: virtio
+    - size: 30GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: localvol_disk
+      interface: virtio
+    state: running
+
+openshift_rhv_vm_manifest:
```

Comment:
I think this is too big for a default installation. I think we should have the default based on https://docs.openshift.com/container-platform/3.10/install/index.html#single-master-multi-node

and either have the additional nodes commented out and reference the OpenShift documentation https://docs.openshift.com/container-platform/3.10/install/index.html#environment-scenarios or add additional defaults.
- Single Master and Node on One System
- Single Master and Multiple Nodes
- Single Master, Multiple etcd, and Multiple Nodes
- Multiple Masters Using Native HA with Co-located Clustered etcd
- Multiple Masters Using Native HA with External Clustered etcd

We need to support  an all-in-one installation if only 1 master node is chosen, for the metrics store.
That will require adding the vm profile that is similler to the master profile but with bigger disk for elasticsearch storage.


 @cwilkers - Can you please share the playbook that will deploy OpenShift?

Reply:
../prerequisites.yml and ../deploy-cluster.yml

Reply:
I thought you are working on generating the openshift inventory file based on the nodes you created.
I generate the openshift inventory file in my playbook, based on the given nodes so I can tailor this together. I only need to check with you about the parameters that I'm missing from my files.

Reply:
I'm not generating an inventory file, just creating the vms data structure and, when I have DNS again, I can add IPs to the inventory hosts' variables.